# This contains a Point (x,y,z) representing the desired target of a robot path
---
geometry_msgs/Point target_pos
